src/lib/definition-parser.ts

```diff
@@ -155,28 +155,25 @@ async function getTypingDataForSingleTypesVersion(
     oldMajorVersion: number | undefined,
 ): Promise<TypingDataFromIndividualTypeScriptVersion> {
     const tsconfig = await fs.readJson("tsconfig.json") as TsConfig; // tslint:disable-line await-promise (tslint bug)
```

Comment:
allReferencedFiles finds all the files that are referenced transitively from the files listed in tsconfig. Then we manually reference d.ts files from OTHER_FILES.txt as references.

Next, the external dependencies are calculated. This part doesn't change too much, I just moved `checkAllFilesUsed`'s call earlier and the no-windows-\ check earlier, into `checkAllFilesUsed`.